components/StayingPopulation.vue

```diff
@@ -13,7 +13,7 @@
         {{ placeName }}
       </div>
       <div class="StayingPopulation-state">
-        [ {{ date }}〜{{ enddate }} ]<br />
+        [ {{ date }}{{ $t('時点') }} ]<br />
```

Comment:
英語で As of xxxx と表示することを考慮して
```suggestion
        [ {{ $t('{date}時点', { date: date }) }} ]<br />
```
とした方がよいと思います．

Reply:
@nard-tech フォローありがとうございます！確かにそうですね。採用させていただきます 🙏